SPECS/grub2/grub2.spec

```diff
@@ -302,9 +309,11 @@ cp $GRUB_MODULE_SOURCE $EFI_BOOT_DIR/$GRUB_MODULE_NAME
 %files efi-binary
 %ifarch x86_64
 /boot/efi/EFI/BOOT/grubx64.efi
+/boot/efi/EFI/BOOT/grubx64-pxe.efi
```

Comment:
Let's not package both grubx64.efi and grubx64-pxe.efi in the same overall grub2-efi-binary RPM. A system will only need one grubx64.efi binary to boot from and having both binaries present but one not active takes up space and can cause potential confusion of which binary is used when we inspect a running system.

Instead consider creating a new subpackage for grubx64-pxe.efi and users who need grubx64-pxe.efi can install that RPM package specifically.

Reply:
Good point. Fixed!